specification/communication/data-plane/Common/readme.md

```diff
@@ -42,11 +42,19 @@ input-file:
 ### Tag: package-2021-03-07-stable
 These settings apply only when `--tag=package-2021-03-07-stable` is specified on the command line.
 
-```yaml $(tag) == 'package-2020-11-19-preview1'
+```yaml $(tag) == 'package-2021-03-07-stable'
 input-file:
   - stable/2021-03-07/common.json
 ```
 
+### Tag: package-2022-07-13-stable
+These settings apply only when `--tag=package-2022-07-13-stable` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-07-13-stable'
+input-file:
```

Comment:
Do you really want to ship a package with only the definitions from the common.json file?

Reply:
@johanste Wonderful question, I'd say no.
The other ACS service swaggers reference the common swagger via relative file path, and the service client SDKs reference their respective README files (used to be direct raw urls to the swagger).

Would you recommend to just delete all README files from the common folder?

Reply:
I can't delete the README files nor the references to the common.json without Avocado failing the CI check :-(